cli/asc.js

```diff
@@ -39,12 +39,9 @@ const colorsUtil = require("./util/colors");
 const optionsUtil = require("./util/options");
 const mkdirp = require("./util/mkdirp");
 const find = require("./util/find");
+const dynrequire = require("./util/dynrequire");
```

Comment:
How about?
```suggestion
require = require("./util/dynrequire");
```
?
But make sure it placed last (after line 43)

Reply:
Hmm, I see that leading to confusion in PRs by someone not familiar with the code base, since it semantically hides what which require does.

Reply:
Why? After recent changes it could be done as:
```ts
if (typeof __webpack_require__ === "function") {
   require = __non_webpack_require__;
}
```

Reply:
As an example, I imagine someone `require`ing something later than that, in their own fork during development or just for testing for instance, and their bundle exploding. Explicitly typing `dynrequire` makes it much more obvious what will happen.

Reply:
alright

Reply:
The inverse might work, though, like us using `webpackRequire` instead of `require` where something should be bundled, making `require` default to the dynamic version.

Reply:
webpack also have differentt mode hins for dynamic importing:

```ts
require(/* webpackMode: "lazy" */ `ts-node`)
require(/* webpackMode: "lazy-once" */ `ts-node`)
```

More details: https://webpack.js.org/api/module-methods

Reply:
`/* webpackIgnore: true */` is nice as well because it doesn't need any actual code